frontend/src/routes.tsx

```diff
@@ -45,17 +48,17 @@ export const routes: RouteItem[] = [
         key: "lights",
         default: false,
     },
-    {
-        route: "/devices/task_monitor",
-        component: <TaskMonitor />,
-        exact: true,
-        icon: <SettingsOutlinedIcon />,
-        category: "Devices",
-        type: routeType.COLLAPSE,
-        name: "Task Monitor",
-        key: "task_monitor",
-        default: false,
-    },
+    // {
```

Comment:
remove?

Reply:
This will be re-added for the beta release.

Reply:
Added documentation for that